src/controllers/BaseController.ts

```diff
@@ -184,8 +184,11 @@ export class BaseController<FORM> extends BaseAsyncHttpController {
 
                 if (that.formActionProcessors != null) {
                     for (const actionProcessorType of that.formActionProcessors) {
-                        const actionProcessor = that.httpContext.container.get(actionProcessorType);
-                        await actionProcessor.process(request, response);
```

Comment:
This was making one of the test changes in PenaltyDetailsController.test.ts fail. Had to change it to the correct type